crypto/evp/evp_test.cc

```diff
@@ -1009,14 +1009,14 @@ TEST(EVPTest, ECTLSEncodedPoint) {
     };
 
     struct ectlsencodedpoint_test_data x25519_test_data = {
-      .public_key = kX25519PublicKey,
```

Comment:
These are all new test vectors and removing the designated initializer follows other standard practice.

Reply:
Apparently, designated initialiser is not supported before C++20. Apparently.
https://en.cppreference.com/w/cpp/language/aggregate_initialization